test/test_filesystem.cpp

```diff
@@ -18,60 +18,79 @@
 #include "c_utilities/filesystem.h"
 
 static char cwd[1024];
+#ifdef WIN32
+const char delimiter = '\\';
+#else
+const char delimiter = '/';
+#endif
 
 TEST(test_filesystem, utilities_exists) {
   utilities_get_cwd(cwd, 1024);
```

Comment:
I guess only `utilities_get_cwd` has arguments which are used to output information from the functions. For the other functions the return value is the only output. I don't think we need the a attribute for those. Sorry for being unclear in the previous comment.